MessageCenter/Classes/Views/Chatting/GroupChannelChattingViewController.swift

```diff
@@ -631,6 +629,33 @@ class GroupChannelChattingViewController: UIViewController, SBDConnectionDelegat
         self.present(mediaUI, animated: true, completion: nil)
     }
     
+    @objc private func sendFileMessage() {
+        let status = PHPhotoLibrary.authorizationStatus()
+        if status == .authorized {
+            openPicker()
+        } else {
+            PHPhotoLibrary.requestAuthorization { status in
+                switch status {
+                case .authorized:
+                    self.openPicker()
+                    break
+                case .denied, .restricted:
+                    DispatchQueue.main.async {
+                        let vc = UIAlertController(title: "Error", message: "Authorization to assets is denied.", preferredStyle: UIAlertControllerStyle.alert)
```

Comment:
Such strings must be localized.

Reply:
Also the Cancel button.

Reply:
we don't implement localization yet. all other messages are not localized

Reply:
As discussed in Slack, I did setup the localization and l localized the Select Location Screen.. new messages should be localized, we'll revisit the whole sdk later to complete any missing translations.